src/FunctionApps/python/IntakePipeline/fhir.py

```diff
@@ -1,8 +1,36 @@
-from typing import Iterator
+import io
+import json
+import logging
 
+from typing import Iterator, IO
 
-def read_fhir_bundles() -> Iterator[dict]:
-    yield {}
+from azure.identity import DefaultAzureCredential
+from azure.storage.blob import ContainerClient
+
+
+def get_blob_client(container_url: str) -> ContainerClient:
```

Comment:
We'll probably want some function descriptor docstrings in all of these functions (i.e

```
def get_blob_client(container_url: str) -> ContainerClient:
'''
Function to blah blah blah
'''
    pass
```